src/main/java/emissary/command/ServerCommand.java

```diff
@@ -99,7 +100,8 @@ public void setupServer() throws EmissaryException {
     @Override
     protected void startService() {
         try {
-            LOG.info("Running Emissary Server");
+            LOG.info("Starting Emissary Server - version: {} - built on {} - git hash: {}", GitRepositoryState.getRepositoryState().getBuildVersion(),
```

Comment:
One last thought - I'm thinking we should move this to a location that is easier to override/update. For instance, we could add a method like dumpVersionInfo and add it after dumpBanner in Emissary.java to get:
```
|  ___|  \/  |_   _/  ___/  ___|/ _ \ | ___ \ \ / /
| |__ | .  . | | | \ `--.\ `--./ /_\ \| |_/ /\ V / 
|  __|| |\/| | | |  `--. \`--. \  _  ||    /  \ /  
| |___| |  | |_| |_/\__/ /\__/ / | | || |\ \  | |  
\____/\_|  |_/\___/\____/\____/\_| |_/\_| \_| \_/  

Emissary version: 7.9.0-SNAPSHOT built on 2022-09-10T10:39:15+0000 git hash: edaa856
```
This way we could output the version info for an implementation pretty easy - something like:
```
\___ /_,___/__,_/_/ /_/ /_/_/____/ ___/

Emissary version: 7.9.0-SNAPSHOT built on 2022-09-10T10:39:15+0000 git hash: edaa856
Implementation version: 0.0.1-SNAPSHOT built on 2022-09-10T11:12:34+0000 git hash: 1234567
``` 
Thoughts?